testsuite/admin-client-tests/pom.xml

```diff
@@ -21,4 +21,16 @@
         </dependency>
     </dependencies>
 
+    <build>
```

Comment:
Perhaps we invert this so that only the artifacts that get published are included explicitly? This avoid accidentally publishing new modules that might be introduced in the future.

Reply:
Inversion of this condition will be even more confusing, due to default value for the property driving this.
I just followed the way how we are using it in keycloak main repo.